QUARKUS-5666.md

```diff
@@ -0,0 +1,43 @@
+# QUARKUS-5666 Add support for encrypted PKCS#8 - technology preview
+
+### Links to the resources
+
+- JIRA ticket: https://issues.redhat.com/browse/QUARKUS-5666
+- GH PR related to the JIRA issue: https://github.com/quarkusio/quarkus/pull/44549
+- Guide modified and related:
+    * https://quarkus.io/guides/tls-registry-reference
+
+## Scope of the testing
+Modification of the Test Framework (TF) to introduce Format.ENCRYPTED_PEM in the Certificate class to use it with the @Certificate 
```

Comment:
```suggestion
Modification of the Test Framework (TF) to introduce `Format.ENCRYPTED_PEM` in the `Certificate` class to use it with the `@Certificate `
```

the `Certificate` should probably be replaced with full class name (including package, not just the simple name)